helm/botkube/values.yaml

```diff
@@ -60,23 +60,6 @@ rbac:
         - apiGroups: ["*"]
           resources: ["*"]
           verbs: ["get", "watch", "list"]
-    'argocd':
```

Comment:
Should we move all those removed configurations to the docs as examples? Since we now have a YAML config editor, it will be easy to copy them over after configuring them through the UI.

Reply:
Yes, good idea. As a part of the documentation task I'll update:
- instructions how to create RBAC group
- examples for plugin configuration

👍